### PR TITLE
Reword glossary entry for kubectl

### LINKED ISSUE #39932: Glossary entry for kubectl also includes lower-case version of name
> Kubectl
Also known as:_kubectl_

it doesn't make sense...

### PR DESCRIPTION
**Pull request by KhushPatibandha**
resolved #39932

### CODE REVIEW

File: content/en/docs/reference/glossary/kubectl.md [hunk 1]
```diff
@@ -7,12 +7,11 @@ short_description: >
   A command line tool for communicating with a Kubernetes cluster.
 
 aka:
```
Comment (sftim) [inline content/en/docs/reference/glossary/kubectl.md:9]:
```suggestion
```
Also remove this line, then.

File: content/en/docs/reference/glossary/kubectl.md [hunk 2]
```diff
@@ -7,12 +7,11 @@ short_description: >
...
-- kubectl
 tags:
 - tool
 - fundamental
 ---
-Command line tool for communicating with a Kubernetes cluster's
+Kubectl (or kubectl) is a command line tool for communicating with a Kubernetes cluster's
```
Comment (sftim) [inline content/en/docs/reference/glossary/kubectl.md:14]: 1. Check that this change doesn't break any pages relying on the existing glossary text.
2. I prefer the original. Do we need to reword?

**Comment by bradtopol** [comment]
Assigning sftim since he is already engaged on this.
/assign sftim

**Comment by sftim** [comment]
/retitle Reword glossary entry for kubectl

I am minded to close this as I prefer the original wording.

**Pull request closed without merging**